internal/terraform/rootmodule/errors.go

```diff
@@ -14,3 +14,11 @@ func (e *RootModuleNotFoundErr) Error() string {
 	}
 	return "root module not found"
 }
+
+func IsRootModuleNotFound(err error) bool {
```

Comment:
These should either wrap or just be replaced by [`errors.Is`](https://godoc.org/errors#Is) but doesn't need to happen in this PR.

Reply:
Happy to address that in a follow-up PR.